docs/eventing/features/authorization.md

```diff
@@ -0,0 +1,471 @@
+    * **Use Case**: Use `from.sub` to allow specific users or service accounts, or to apply wildcard patterns for more flexibility.
+    ```yaml
+    from:
+      - sub: system:serviceaccount:default:trusted-app
+      - sub: "system:serviceaccount:default:other-*"
+    ```
+
+### Advanced CloudEvent filtering criterias
+
+The `.spec.filters` section is optional and specifies additional criteria that the event itself must meet to be allowed. 
+
+* **Example:** Only CloudEvents with the `type` equals to `com.github.push` and matching a certain CESQL expression are allowed.
+* **Use Case:** Use `filters` when you want to have more fine grained criterias on allowed CloudEvents.
+  ```yaml
+  from:
+    filters:
+      - cesql: "type IN ('order.created', 'order.updated', 'order.canceled')"
+      - exact:
+          type: com.github.push
+  ```
+
+If the filters are specified, an event must match **all** the specified filters of an `EventPolicy` (in addition to its `.spec.from`) to be accepted. `.spec.filters` accepts the same filter dialects as [Triggers](../triggers/README.md#supported-filter-dialects).
+
+!!! note
+    Filters apply in addition to `.spec.from`. This means, soon as an `EventPolicy` specifies `.spec.filters`, they must match the request, as well as the `.spec.from` (*AND* operand). Only then the `EventPolicy` allows the request.
+
+### Summary of `.spec` fields:
+
+* `to.ref` targets a specific resource.
+* `to.selector` targets a set of resources based on labels.
+* `from.ref` authorizes a specific event source resource.
+* `from.sub` authorizes specific users, service accounts, or patterns of accounts.
+* `.spec.filters` allows to define advanced CloudEvent filtering criterias
+
+## EventPolicy status
+
+The status of an `EventPolicy` provides information about resolved sources and readiness:
+
+```yaml
+status:
+  from:
+    - system:serviceaccount:default:my-source-oidc-sources.knative.dev-pingsource
+    - system:serviceaccount:default:trusted-app
+    - "system:serviceaccount:default:other-*"
+  conditions:
+    - type: Ready
+      status: "True"
+    - type: SubjectsResolved
+      status: "True"
+```
+
+## Applying EventPolicies to Resources
+
+Event consumers, such as a `Broker`, will list the applying EventPolicies in their status:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: my-broker
+spec:
+  ...
+status:
+  ...
+  policies:
+    - name: my-event-policy
+      apiVersion: v1alpha1
+  conditions:
+    - type: Ready
+      status: "True"
+    - type: EventPoliciesReady
+      status: "True"
+```
+
+The `EventPoliciesReady` condition indicates whether all applicable EventPolicies for a resource are ready and have been successfully applied.
+
+## Rejection Behavior
+
+If a request does not pass any applicable `EventPolicy`, it will be rejected with a `403 Forbidden` HTTP status code, ensuring that unauthorized event deliveries are blocked.
```

Comment:
I think it's currently implied here, but maybe we could make it clear that when multiple policies apply to the same resource, if the event matches one of the policies the event will be delivered?

Reply:
Good point to make it clearer :+1: Updated it in 993c2a39a6226ca92122df6dfdf9e02f30d4df99